tests/integration/targets/postgresql_ping/tasks/postgresql_ping_initial.yml

```
@@ -46,6 +46,39 @@
       - result.server_version == {}
       - result is not changed
 
+- name: postgresql_ping - check ping of the database on non-existent port does not return anything
+  become_user: "{{ pg_user }}"
+  become: true
+  postgresql_ping:
+    db: "{{ db_default }}"
+    login_user: "{{ pg_user }}"
+  register: result
+  environment:
+    PGPORT: 5435
+  ignore_errors: true
```

```suggestion
  environment:
    PGPORT: 5435
  ignore_errors: true
  postgresql_ping:
    db: "{{ db_default }}"
    login_user: "{{ pg_user }}"
  register: result
```
i dunno about anyone else., but when I read a task, I like to see the 'supporting info' first since in my mind it works that way (set these env vars, ignore errors, run this)

---

if you agree, do the same for the other tasks pls